lib/middleware/websocket/client.rb

```diff
@@ -1,10 +1,10 @@
 module Websocket
   class Client
-    attr_reader :connection_client, :client_attributes
+    attr_reader :connection, :client_attributes
     attr_accessor :position
 
-    def initialize(connection_client: client)
-      @connection_client = connection_client
+    def initialize(connection: client)
```

Comment:
This default value is undefined, as far as I can tell.

Reply:
@jgayfer fixed now, will adjust going forward.